lib/debug/server_dap.rb

```diff
@@ -655,6 +659,49 @@ def process_protocol_request req
       end
     end
 
+    def request_rdbgTraceInspector(req)
+      cmd = req.dig('arguments', 'command')
+      case cmd
+      when 'enable'
+        events = req.dig('arguments', 'events')
+        evts = []
+        events.each{|evt|
```

Comment:
Another idea:

```ruby
EVENTS = {'line' => [:line], ...}

def ...
  evts = []
  events.each{|evt| evts.concat EVENTS[evt]}
  ...
```

or 

```ruby
  evts = events.flat_map{|evt| EVENTS[evt]}
```

But the current code makes sense because it is easy to read.